docs/replatforming.md

```diff
@@ -84,4 +88,21 @@ The general steps for migrating a guide:
     ```
 6. Celebrate! Or edit this documentation to update any steps that may be missing.
 
+## Running pa11y
+We use `pa11y-ci` is used to scan for accessibility issues. The scan runs as part of
+our CI setup (see the [pull-request.yml workflow](.github/workflows/pull-request.yml))
+on every pull request, but it can also be run locally. To run locally, type:
+
+```
+npm run test:pa11y-ci
+```
+
+Note that running `pa11y-ci` inside the docker container may not always work.
 
+In cases where you want pa11y to ignore a certain element, such as in the accessibility guide which intentionally shows examples of accessibility issues, you can add the data attribute `data-pa11y-ignore` to the element that should be ignored.
```

Comment:
do we need to document when we use this `data-pa11y-ignore` ? I have noticed there are some field that use it, but I did not see the reason we use them.

Reply:
It wasnt as clear as compared to your comment above (much more like plain text language) :) May want to rephrase it, or use the one you just put here.